weblate_web/templates/snippets/reviews.html

```diff
@@ -0,0 +1,73 @@
+{% load i18n %}
+{% load static %}
+
+<div>
+    <a href=""><img src="{% static "img/Siemens-logo.png" %}" alt="Siemens" /></a>
+    <a href=""><img src="{% static "img/Dyson-logo.png" %}" alt="Dyson" /></a>
+    <a href=""><img src="{% static "img/YouGov-logo.png" %}" alt="YouGov" /></a>
+ </div>
+ <br>
+ <div class="simple-slider page-slider" dir="ltr">
+    <div class="slider-wrapper" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}">
+       <div class="slider-slide is-bright">
+          <div class="testimonial">
+             <div class="icon-review">
+                <img src="{% static "img/Quotes.png" %}" alt="Quotes" />
+                <p>Weblate is a gorgeous app! It really saved my life. Translations are managed within same version control as the code. Weblate is a gorgeous app! It really is my life. Translations are managed within it.</p>
```

Comment:
```suggestion
                <p>Weblate is gorgeous! It really saved my life. Translations are managed within the same version-control system as the code. Weblate is gorgeous! It really is my life. Translations are managed within it.</p>
```

Reply:
These are placeholders until we get real quotes.